spec/velveteen/parse_message_spec.rb

```diff
@@ -6,15 +6,15 @@
     body = '{"foo": "bar"}'
     delivery_info = double
     headers = double
-    properties = double(headers: headers)
+    properties = double("headers" => headers)
```

Comment:
This is stubbing a method call, so symbol/string doesn't matter. Probably more idiomatic to use a symbol, though.